build/Targets/Imports.targets

```diff
@@ -329,7 +329,7 @@
   <!-- This is a copy of the Microsoft.VisualStudio.SDK.EmbedInteropTypes NuGet package, but only the list of
        assemblies that we need. The package includes things like EnvDTE which are reasonable for consumers, but
        strange since we actually _implement_ DTE and use it as an exchange type with generics in a few places. -->
-  <Target Name="LinkVSSDKEmbeddableAssemblies" DependsOnTargets="ResolveReferences" AfterTargets="ResolveReferences">
+  <Target Name="LinkVSSDKEmbeddableAssemblies" BeforeTargets="FindReferenceAssembliesForReferences" >
```

Comment:
Not according to @rainersigwald:

> @davkean You can work around for the moment by adding a BeforeTargets="FindReferenceAssembliesForReferences" to that target. EDIT: wait, that probably creates a dependency cycle. You probably have to replace the dependencies with that rather than add it.